contrib/kind-common.sh

```
@@ -1553,23 +1552,16 @@ clone_frr() {
     git checkout --detach "$FRR_K8S_GIT_REF"
     popd
 
-    # Download the patches
-    curl -Ls https://github.com/jcaamano/frr-k8s/archive/refs/heads/ovnk-bgp-v0.0.21.tar.gz | tar xzvf - frr-k8s-ovnk-bgp-v0.0.21/patches --strip-components 1
-
-    # Change into the cloned repo directory before applying patches
     pushd frr-k8s
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)